demo-app/src/components/movie-form/MovieForm.css

```
@@ -0,0 +1,92 @@
+.movie-form-container {
+    width: 976px;
+    height: 917px;
+    background-color: #232323;
+    display: flex;
+    padding: 60px;
+    gap: 30px;
+    flex-direction: column;
+    background: #232323;
+}
+
+.movie-form-row {
+    display: flex;
+    flex-wrap: nowrap;
+    gap: 30px;
+    color: #FFF;
+    font-family: Montserrat;
+    font-size: 20px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: normal;
+}
+
+.movie-form-label {
+    opacity: 0.8px;
```

Mismatched property value (number | percentage)
```suggestion
    opacity: 0.8;
```
Fix here and below in other rules